### PR TITLE
Add chip selection ui

### PR DESCRIPTION
**Pull request by macandcheese**
Adds dummy multi-chip select - still needs source data.
Updates some UI and styles.

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -68,10 +69,17 @@ async function init() {
       block.appendChild(image);
 
       if (attributes["WEBSITE"]) {
-        const websiteChip = document.createElement("calcite-chip");
-        websiteChip.id = "detail-chip-website";
-        websiteChip.innerText = `website: ${attributes["WEBSITE"]}`;
-        block.appendChild(websiteChip);
+        const itemWebsite = document.createElement("calcite-button");
+        itemWebsite.setAttribute("id", "detail-item-website");
```
Comment (driskull) [inline app/app.js]: nitpick: I don't think we need to call setAttribute. We can just set the property.

File: app/app.js [hunk 2]
```diff
@@ -90,24 +98,26 @@ async function init() {
...
+
       document.getElementById(
         "detail-chip-pop"
       ).innerText = `population: ${attributes["POPULATION"]}`;
     }
     view.goTo(
       {
         center: [result.geometry.longitude, result.geometry.latitude],
-        zoom: 13,
+        zoom: 10,
```
Comment (driskull) [inline app/app.js]: Julie wanted this at 13 last time we spoke. Is this ok @juliepowell ?

**Review by driskull — APPROVED** [review]

File: app/app.js (hunk 2 above)
Comment (macandcheese) [reply app/app.js]: Happy to revert.

File: app/app.js (hunk 1 above)
Comment (macandcheese) [reply app/app.js]: Updated.

**Review by driskull — APPROVED** [review]

**Comment by macandcheese** [comment]
Re-requested review after adding filtering logic.

File: app/app.js [hunk 3]
```diff
@@ -70,10 +71,17 @@ async function init() {
       block.appendChild(image);
 
       if (attributes["WEBSITE"]) {
-        const websiteChip = document.createElement("calcite-chip");
-        websiteChip.id = "detail-chip-website";
-        websiteChip.innerText = `website: ${attributes["WEBSITE"]}`;
-        block.appendChild(websiteChip);
+        const itemWebsite = document.createElement("calcite-button");
+        itemWebsite.id = "detail-item-website";
+        itemWebsite.setAttribute("icon-end", "launch");
```
Comment (driskull) [inline app/app.js]: Can we get rid of the `setAttribute`s? :)

File: app/app.js [hunk 4]
```diff
@@ -396,6 +401,33 @@ async function init() {
     queryItems();
   });
 
+  // Degree type chip select
+  for (const [key, value] of Object.entries(appConfig.programTypes)) {
+    const chip = document.createElement("calcite-chip");
+    chip.setAttribute("tabindex", "0");
```
Comment (driskull) [inline app/app.js]: moar setattributes :)

**Review by driskull — APPROVED** [review]
👍 💯 

Just cleanup of setAttribute :)

**Pull request merged**